src/styles/index.scss

```diff
@@ -2,2 +1,152 @@
+.body {
   margin: 0;
 }
+
+.button {
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  margin: 16px;
+  background: #00acdc;
+  border: 0;
+  align-items: center;
+  color: #fff;
+  font-family: Roboto, sans-serif;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+
+  &:hover {
+    background: #fff;
+    border: 1px solid #00acdc;
+    color: #00acdc;
+  }
+}
+
+.card {
+  width: 199px;
+  height: 100%;
+  border: 1px solid rgb(240, 7, 228);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  &__image {
+    box-sizing: border-box;
+    background-image: url(/src/images/imac.jpeg);
+    background-size: cover;
+    width: 160px;
+    height: 134px;
+    margin-top: 33px;
+    margin-left: 20px;
+  }
+
+  &__name {
+    width: 166px;
+    height: 36px;
+    padding: 40px 16px 0;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+  }
+
+  &__code {
+    width: 98px;
+    height: 14px;
+    padding: 4px 16px;
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    color: #616070;
+  }
+
+  &:hover {
+    border: 1px solid blue;
+    border-radius: 5px;
+  }
+
+  &__price {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 24px 16px 0;
+  }
+
+  &__price-key {
+    width: 31px;
+    height: 18px;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    text-align: left;
+    color: #616070;
+  }
+
+  &__price-value {
+    width: 50px;
+    height: 18px;
+    font-family: Roboto, sans-serif;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    text-align: right;
+    color: #060b35;
+  }
+}
+
+.review {
+  display: flex;
+  flex-direction: row;
+  padding: 16px 16px 0;
+  justify-content: space-between;
+  align-items: center;
+
+  &__reviews {
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #060b35;
+  }
+}
+
+.stars {
+  height: 16px;
+  display: flex;
+  flex-direction: row;
+
+  &__star-active {
```

Comment:
The `align-items` property is used here, but it will not have any effect since the parent container does not have `display: flex` or `display: grid`. If the intention is to center the background image, you should use `background-position: center;` which is already being used.